src/Microsoft.TestPlatform.Common/Resources/Resources.resx

```diff
@@ -201,6 +204,9 @@
   <data name="TestCaseFilterFormatException" xml:space="preserve">
     <value>Incorrect format for TestCaseFilter {0}. Specify the correct format and try again. Note that the incorrect format can lead to no test getting executed.</value>
   </data>
+  <data name="VSInstallationNotFound" xml:space="preserve">
+    <value>Unable to find devenv under path {0}. Make sure tests are run via vstest.console.exe installed as part of Visual Studio.</value>
```

Comment:
Recommand `/TestAdapterPath` here.

Reply:
Is it okay to mention `devenv`, this is user facing text I believe. There are two vstest.console.exe, which one are we recommending to the user?